### PR TITLE
[Ready] More riot gear, cap pen and more!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds in a crate for Capo pen.
Cutlery set crate for cooks and the like
Riot shotguns 
Lizard and Cat stuff and more
[LEWDSTUFFS] *Oh No*
[why]
It just felt like the right thing to do!

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
https://cdn.discordapp.com/attachments/499347891161137172/508452878260764687/unknown.png
For proof

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 			new item(C)
 
 // If you add something to this list, please group it by type and sort it alphabetically instead of just jamming it in like an animal
+//A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P, Q, R, S, T, U, V, W, X, Y, Z <-- For quick and easy references
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: Congrats, you understand the alphabet I guess??

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -2182,6 +2219,55 @@
...
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
+	access = ACCESS_CAPTAIN
+	cost = 25000
+	contains = list(/obj/item/pen/fountain/captain)
+	crate_name = "captain pen"
+	crate_type = /obj/structure/closet/crate/secure/weapon //It is a combat pen
+
+/datum/supply_pack/misc/captain_headset
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: Why would you allow people to buy all-access radio keys

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -2182,6 +2219,55 @@
...
+	name = "Captain Headset Key"
+	desc = "A spare Captain headset key, just in case The Captain ''lost'' it again." 
+	access = ACCESS_CAPTAIN
+	cost = 20000
+	contains = list(/obj/item/encryptionkey/heads/captain)
+	crate_name = "captain headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/cargo_headsets
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: These would be better as a department protolathe thing instead of cargo pack bloat, if anything.

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -2182,6 +2219,55 @@
...
+	cost = 2500
+	contains = list(/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo,
+					/obj/item/encryptionkey/headset_cargo)
+	crate_name = "cargo headset keys"
+	crate_type = /obj/structure/closet/crate/secure
+
+/datum/supply_pack/misc/ce_headset
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: Why should head access keys be purchasable?

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -2182,6 +2219,55 @@
...
+	name = "Chief Engineer Headset Key"
+	desc = "A spare CE headset key. For when the CE throws there key into the SM again."
+	access = ACCESS_CE
+	cost = 10000
+	contains = list(/obj/item/encryptionkey/heads/ce)
+	crate_name = "chief engineer headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/cmo_headset
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: Why should head access keys be purchasable?

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -2271,6 +2370,72 @@
 	crate_name = "coffin"
 	crate_type = /obj/structure/closet/crate/coffin
 
+/datum/supply_pack/misc/hos_headset
```
Comment (Poojawa) [inline code/modules/cargo/packs.dm]: Why should head access keys be purchasable?

**Review by Poojawa — COMMENTED** [review]
etc etc etc

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: It helped me a bit.

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Borgs, traitors or lings.

File: code/modules/cargo/packs.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: When Heads loos there headsets in the SM/Singulo/tesla/space

File: code/modules/cargo/packs.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: For when the CMO headset keys are all stolen by Robots

File: code/modules/cargo/packs.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: For lings/Traitors/silicons.

File: code/modules/cargo/packs.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Ok fair

File: code/modules/cargo/packs.dm [hunk 7]
```diff
@@ -2182,6 +2219,42 @@
...
+	cost = 25000
+	contains = list(/obj/item/pen/fountain/captain)
+	crate_name = "captain pen"
+	crate_type = /obj/structure/closet/crate/secure/weapon //It is a combat pen
+
+/datum/supply_pack/misc/captain_headset
+	name = "Captain Headset Key"
+	desc = "A spare Captain headset key, just in case The Captain ''lost'' it again." 
+	access = ACCESS_CAPTAIN
+	cost = 20000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (silicons) [inline code/modules/cargo/packs.dm]: no get rid of this

File: code/modules/cargo/packs.dm [hunk 8]
```diff
@@ -1541,6 +1551,33 @@
...
+					/obj/item/trash/plate,
+					/obj/item/trash/plate,
+					/obj/item/trash/plate,
+					/obj/item/reagent_containers/food/drinks/drinkingglass,
+					/obj/item/reagent_containers/food/drinks/drinkingglass,
+					/obj/item/reagent_containers/food/drinks/drinkingglass,
+					/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass,
+					/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass)
+	crate_name = "kitchen cutlery deluxe set"
+	dangerous = TRUE //We want Admins to know of this
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: we don't care.

File: code/modules/cargo/packs.dm [hunk 9]
```diff
@@ -519,6 +520,15 @@
 					/obj/item/shield/riot)
 	crate_name = "riot shields crate"
 
+/datum/supply_pack/security/armory/riotshotguns
+	name = "Riot Shotgun Crate"
+	desc = "For when the greytide gets really uppity. Contains three riot Shotguns. Requires Armory access to open."
+	cost = 6000
```
Comment (silicons) [inline code/modules/cargo/packs.dm:525]: how much do combat shotguns cost again i forgot

File: code/modules/cargo/packs.dm [hunk 10]
```diff
@@ -1541,6 +1551,33 @@
 	crate_name = "beekeeping starter crate"
 	crate_type = /obj/structure/closet/crate/hydroponics
 
+/datum/supply_pack/organic/cutlery
+	name = "Kitchen Cutlery Deluxe Set"
+	desc = "Need to slice and dice away those ''Tomatos'' well we got what you need! Form a nice set of knifes, forks, plates, glasses, and a whetstone for when you got some grizzle that is a bit harder to slice then normal."
+	cost = 25000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no what whetstones are powerful but they're not 25k

File: code/modules/cargo/packs.dm [hunk 11]
```diff
@@ -2182,6 +2219,42 @@
 	crate_name = "art supply crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
+	access = ACCESS_CAPTAIN
+	cost = 25000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no what no you don't pay 25k for a pen

File: code/modules/cargo/packs.dm [hunk 12]
```diff
@@ -2182,6 +2219,42 @@
...
+	cost = 20000
+	contains = list(/obj/item/encryptionkey/heads/captain)
+	crate_name = "captain headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/ce_headset
+	name = "Chief Engineer Headset Key"
+	desc = "A spare CE headset key. For when the CE throws there key into the SM again."
+	access = ACCESS_CE
+	cost = 10000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no

File: code/modules/cargo/packs.dm [hunk 13]
```diff
@@ -2182,6 +2219,42 @@
...
+	contains = list(/obj/item/encryptionkey/heads/ce)
+	crate_name = "chief engineer headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/cmo_headset
+	name = "Chief Medical Officer Headset Key"
+	desc = "A spare CMO headset key, For when the CMO's headset gets stollen well they do Genetics."
+	access = ACCESS_CMO
+	cost = 10000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no, just make it a secure headset crate locked to cpatain or something, honestly this seems like a horrible idea

File: code/modules/cargo/packs.dm [hunk 14]
```diff
@@ -2271,6 +2344,47 @@
 	crate_name = "coffin"
 	crate_type = /obj/structure/closet/crate/coffin
 
+/datum/supply_pack/misc/hos_headset
+	name = "Head Of Security Headset Key"
+	desc = "A spare HOS headset key, For when the HOS's headset is being used by a Changling."
+	access = ACCESS_HOS
+	cost = 10000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no

File: code/modules/cargo/packs.dm [hunk 15]
```diff
@@ -2271,6 +2344,47 @@
...
+	contains = list(/obj/item/encryptionkey/heads/hos)
+	crate_name = "head of security headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/hop_headset
+	name = "Head Of Personnel Headset Key"
+	desc = "A spare HOP headset key, For when the HOP's headset is being used by a Traitor."
+	access = ACCESS_HOP
+	cost = 10000
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no

File: code/modules/cargo/packs.dm [hunk 16]
```diff
@@ -2271,6 +2344,47 @@
...
+	contains = list(/obj/item/encryptionkey/heads/hop)
+	crate_name = "head of personnel headset key"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+
+/datum/supply_pack/misc/medsci_headsets
+	name = "Med Sci Headset Keys"
+	desc = "Three Headset Keys for ''Genetics''. To be yelled at for not cloning somone just to be yelled at for cloning them! Also get yelled at for being in sci, well having access."
+	access =  ACCESS_GENETICS
+	cost = 7500
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no..

File: code/modules/cargo/packs.dm [hunk 17]
```diff
@@ -2271,6 +2344,47 @@
...
+	contains = list(
+					/obj/item/encryptionkey/headset_medsci,
+					/obj/item/encryptionkey/headset_medsci,
+					/obj/item/encryptionkey/headset_medsci)
+	crate_name = "med sci headset keys"
+	crate_type = /obj/structure/closet/crate/secure
+
+/*Todo Add in a QM headset
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: no!

File: code/modules/cargo/packs.dm [hunk 18]
```diff
@@ -2284,6 +2398,36 @@
...
+					/obj/item/electropack/shockcollar,
+					/obj/item/restraints/handcuffs/fake/kinky,
+					/obj/item/restraints/handcuffs/fake/kinky,
+					/obj/item/clothing/head/kitty/genuine, // Why its illegal
+					/obj/item/clothing/head/kitty/genuine,
+					/obj/structure/reagent_dispensers/keg/aphro)
+	crate_name = "lewd kit"
+	crate_type = /obj/structure/closet/crate
+
+datum/supply_pack/misc/lewdkeg
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: you forgot a /

File: code/modules/cargo/packs.dm [hunk 19]
```diff
@@ -2284,6 +2398,36 @@
 					)
 	crate_name = "religious supplies crate"
 
+datum/supply_pack/misc/lewd
```
Comment (silicons) [inline code/modules/cargo/packs.dm]: /

**Review by silicons — DISMISSED** [review]
fix it

File: code/modules/cargo/packs.dm (hunk 9 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm:525]: 8000, + banisters. This is about half of what they cost - the other items Its a good middle ground I think

File: code/modules/cargo/packs.dm (hunk 10 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Its also do to meat cleavers and knifes being high in metal

File: code/modules/cargo/packs.dm (hunk 11 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: sure

File: code/modules/cargo/packs.dm (hunk 12 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Dont want them to cheap

File: code/modules/cargo/packs.dm (hunk 13 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: OH okie

File: code/modules/cargo/packs.dm (hunk 16 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: 5000 good?

File: code/modules/cargo/packs.dm [hunk 20]
```diff
@@ -2182,6 +2217,24 @@
...
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
+	access = ACCESS_CAPTAIN
+	cost = 10000
+	contains = list(/obj/item/pen/fountain/captain)
+	crate_name = "captain pen"
+	crate_type = /obj/structure/closet/crate/secure/weapon //It is a combat pen
+
+/datum/supply_pack/misc/captain_headset
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Ditto for what Poojawa said here. Why is this necessary when traitors and lings are already more than capable of getting all-access radio keys on their own without help from cargo.

File: code/modules/cargo/packs.dm [hunk 21]
```diff
@@ -2182,6 +2217,24 @@
 	crate_name = "art supply crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: s/Captaun/Captain

File: code/modules/cargo/packs.dm [hunk 22]
```diff
@@ -2284,6 +2349,48 @@
...
+/datum/supply_pack/misc/lewdkeg
+	name = "Lewd Deluxe Keg" 
+	desc = "That other stuff not getting you ready? Well I have a Chemslut making tons of the good stuff."
+	cost = 7000 //It can be a weapon
+	contraband = TRUE
+	contains = list(/obj/structure/reagent_dispensers/keg/aphro/strong)
+	crate_name = "deluxe keg"
+	crate_type = /obj/structure/closet/crate
+
+/datum/supply_pack/misc/head_keys
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Ditto for the above comment

**Review by deathride58 — DISMISSED** [review]

File: code/modules/cargo/packs.dm (hunk 20 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Also borgs and standing caps, people tend to take the keys and cryo.

File: code/modules/cargo/packs.dm (hunk 21 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: hashtag good at being a human

File: code/modules/cargo/packs.dm (hunk 22 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: people cryo with them all the time and then "Woops" someone one dies and there key's gone and or headset. This is for that or for borgs

File: code/modules/cargo/packs.dm [hunk 23]
```diff
@@ -1541,6 +1550,32 @@
 	crate_name = "beekeeping starter crate"
 	crate_type = /obj/structure/closet/crate/hydroponics
 
+/datum/supply_pack/organic/cutlery
+	name = "Kitchen Cutlery Deluxe Set"
+	desc = "Need to slice and dice away those ''Tomatos'' well we got what you need! Form a nice set of knifes, forks, plates, glasses, and a whetstone for when you got some grizzle that is a bit harder to slice then normal."
```
Comment (nik707) [inline code/modules/cargo/packs.dm]: Form a nice set of knifes > from?

Comment (nik707) [reply code/modules/cargo/packs.dm]: same line, then should be than

File: code/modules/cargo/packs.dm [hunk 24]
```diff
@@ -2271,6 +2324,18 @@
 	crate_name = "coffin"
 	crate_type = /obj/structure/closet/crate/coffin
 
+/datum/supply_pack/misc/medsci_headsets
+	name = "Med Sci Headset Keys"
+	desc = "Three Headset Keys for ''Genetics''. To be yelled at for not cloning somone just to be yelled at for cloning them! Also get yelled at for being in sci, well having access."
```
Comment (nik707) [inline code/modules/cargo/packs.dm]: Three Headset Keys for ''Genetics''. To be yelled at for not cloning someone, only to be yelled at again for cloning them! Also get yelled at for being in sci, or at least having access.

File: code/modules/cargo/packs.dm [hunk 25]
```diff
@@ -2284,6 +2349,48 @@
...
+	desc = "That other stuff not getting you ready? Well I have a Chemslut making tons of the good stuff."
+	cost = 7000 //It can be a weapon
+	contraband = TRUE
+	contains = list(/obj/structure/reagent_dispensers/keg/aphro/strong)
+	crate_name = "deluxe keg"
+	crate_type = /obj/structure/closet/crate
+
+/datum/supply_pack/misc/head_keys
+	name = "Head Of Staft Headset Keys"
+	desc = "A spare set of keys for heads of staft."
```
Comment (nik707) [inline code/modules/cargo/packs.dm]: staft? It's gonna be Staff. Desc and name.

**Review by nik707 — CHANGES_REQUESTED** [review]
If this stuff gets fixed, and both @deathride58 and @kevinz000 confirm their changes have been done to their own approval, it's ready.

**Review by nik707 — APPROVED** [review]
